lib/kafo/kafo_configure.rb

```diff
@@ -193,13 +193,18 @@ def execute
         @progress_bar = config.app[:colors] ? ProgressBars::Colored.new : ProgressBars::BlackWhite.new
       end
 
-      unless skip_checks_i_know_better?
-        unless SystemChecker.check
-          puts "Your system does not meet configuration criteria"
+      if checks_only? || !skip_checks_i_know_better?
+        logger = Logger.new('checks')
+        if !SystemChecker.check
+          logger.error("Your system does not meet configuration criteria")
           self.class.exit(:invalid_system)
+        else
+          logger.notice("System checks passed")
```

Comment:
I'd argue this is `info`, not `notice`. https://en.wikipedia.org/wiki/Syslog#Severity_level says for notice:
> Conditions that are not error conditions, but that may require special handling.

And info:
> Confirmation that the program is working as expected.

Reply:
The problem we always have is that we had to use NOTICE because of how puppet uses INFO and that forced our hand to break some of these rules in order to provide users with useful information. Notice is our default, and I believe that this is a useful output for default state, especially when running checks only or the output feels like nothing actually happened:

```
2024-04-18 14:08:27 [NOTICE] [root] Loading installer configuration. This will take some time.
2024-04-18 14:08:32 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2024-04-18 14:08:32 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
2024-04-18 14:08:33 [NOTICE] [checks] System checks passed
```

vs.

```
2024-04-18 14:08:27 [NOTICE] [root] Loading installer configuration. This will take some time.
2024-04-18 14:08:32 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2024-04-18 14:08:32 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
```